config/app.default.php

```diff
@@ -244,6 +249,8 @@
              * which is the recommended value in production environments
              */
             //'init' => ['SET GLOBAL innodb_stats_on_metadata = 0'],
+            
+            'url' => env('DATABASE_URL', ''),
```

Comment:
Let me chime in. Here is an example scenario.

Running multiple processes in a Docker container including nginx, php-fpm, etc. I'm passing the env vars outside from Docker. `DEBUG` is too generic and it may affect other process other than CakePHP.